_posts/2015-12-13-documentation-theme-jekyll.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Documentation Theme for Jekyll
+date: 2015-12-13 12:55
+homepage: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/home.html
+download: https://github.com/tomjohnson1492/documentation-theme-jekyll/archive/gh-pages.zip
+demo: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/home.html
+author: tomjohnson1492
+thumbnail: docthemejekyll.png
```

Comment:
Hello and thanks for the PR! Please match the thumbnail name with post name: `documentation-theme-jekyll.png`. Thanks!